hazelcast/src/main/java/com/hazelcast/cluster/impl/ClusterServiceImpl.java

```diff
@@ -919,28 +952,6 @@ public void merge(final Address newTargetAddress) {
         }
     }
 
-    private <V> V waitOnFutureInterruptible(Future<V> future, long timeout, TimeUnit timeUnit)
```

Comment:
Moved this method down to the bottom into the `private class MergeTask` where it is used.